tests/Server/Workspace/SymbolTest.php

```diff
@@ -58,9 +58,10 @@ public function testEmptyQueryReturnsAllSymbols()
             new SymbolInformation('TestInterface',      SymbolKind::INTERFACE,   $this->getDefinitionLocation('TestInterface'),                                ''),
             new SymbolInformation('test_function',      SymbolKind::FUNCTION,    $this->getDefinitionLocation('test_function()'),                              ''),
             new SymbolInformation('ChildClass',         SymbolKind::CLASS_,      $this->getDefinitionLocation('ChildClass'),                                   ''),
+            new SymbolInformation('define',             SymbolKind::CONSTANT,    $this->getDefinitionLocation('TEST_PROPERTY'),                                ''),
```

Comment:
What do you mean with value? A SymbolInformation has a name, and the name of the defined constant is TEST_PROPERTY. What value?